components/brave_wallet_ui/common/slices/endpoints/transaction.endpoints.ts

```diff
@@ -778,7 +778,11 @@ export const transactionEndpoints = ({
     }),
 
     approveTransaction: mutation<
-      { success: boolean },
+      {
+        status: boolean
```

Comment:
can we rename this to `success`?
```suggestion
        success: boolean
```